### PR TITLE
fix: remove user stories from landing page and readme

### PR DESCRIPTION
**Pull request by moT01**
Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39721

Wait until PR on the main repo hits production to merge this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,2 @@
...
-5. If the date string is **invalid** the api returns a JSON having the structure `{"error" : "Invalid Date" }`.
-
-#### Example usage:
-* https://Timestamp-Microservice--freecodecamp.repl.co/api/timestamp/2015-12-25
-*https://Timestamp-Microservice--freecodecamp.repl.co/api/timestamp/1451001600000
-
-#### Example output:
-* {"unix":1451001600000, "utc":"Fri, 25 Dec 2015 00:00:00 GMT"}
\ No newline at end of file
+# [Timestamp Microservice](https://www.freecodecamp.org/learn/apis-and-microservices/apis-and-microservices-projects/timestamp-microservice)
```
Comment (Sky020) [inline README.md:2]:
```suggestion
# [APIs and Microservices Projects - Timestamp Microservice](https://www.freecodecamp.org/learn/apis-and-microservices/apis-and-microservices-projects/timestamp-microservice)
```

**Review by Sky020 — CHANGES_REQUESTED** [review]
Feel free to dismiss this, but either:
1) We have it standard throughout with the block name in the title
2) We change some of the previous PRs to just have the project name

I might favour **2** (without this change), because, then a change of superblock would not require an update of the boilerplate READMEs. What do you think?

File: README.md (hunk 1 above)
Comment (moT01) [reply README.md:2]: I vote for 2 as well, that's how it was in Kris's PR I linked somewhere in this web of PR's - so that's how I've been making them.

**Review by Sky020 — APPROVED** [review]
This LGTM 👍 

Nice catch on the CSS, BTW

**Comment by RandellDawson** [comment]
@moT01 I am going to go ahead and merge this one as there are no changes that would affect the tests or instructions on `/learn`.  I also validated the changes to the `index.html` and `styles.css` match those on the example project.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**